### DISCUSSION TITLE
url_background set as poster?

### DISCUSSION THREAD

**Discussion by r3draid3r04** [meisnate12/Plex-Meta-Manager#1559] [Q&A]
I am trying to update metadata for movies. Setting movie posters and backgrounds for the movies, however, when I set url_background it sets the background image to the poster image. I check the assets folder and it has both the background.jpg and the poster.jpg. Am I doing something wrong? this is my metadata file for a single movie.

`metadata:         

  "Warning (2021)":
    title: "Warning"
    year: 2021
    url_poster: https://theposterdb.com/api/assets/185732
    url_background: https://www.themoviedb.org/t/p/original/x6YTKawdsxjE4xVdyFfvnwkwp24.jpg`

Any help would be greatly appreciated. Thanks!

**Comment by gitttt54** [comment]
that might have been fixed with the latest release, I did not happen again.
